src/openrct2/scripting/bindings/game/ScConfiguration.hpp

```diff
@@ -199,13 +199,8 @@ namespace OpenRCT2::Scripting
                 if (key == "general.language")
                 {
                     auto& localisationService = GetContext()->GetLocalisationService();
-                    auto language = localisationService.GetCurrentLanguage();
-                    auto locale = "";
-                    if (language >= 0 && static_cast<size_t>(language) < std::size(LanguagesDescriptors))
-                    {
-                        locale = LanguagesDescriptors[language].locale;
-                    }
-                    duk_push_string(ctx, locale);
+                    auto locale = localisationService.GetCurrentLanguageLocale();
```

Comment:
This change does not appear to be related to this PR, could you clarify?

Reply:
The method I created to get the current locale is used by asset pack window. The other place this should be used is this plugin code.